### ISSUE TITLE
Command-line generator attempts to include any trailing comments as include file names

### ISSUE THREAD

**Issue by ageoghegan** [mulesoft-labs/raml-dotnet-tools#142]
I have external RAML files referenced in a relative path like so:
```
uses:
  Requests: library/requests.raml
  Responses: library/responses.raml
  Base: library/baseTypes.raml
```
This is working fine now, however when the command line generator tries to parse the included files I get this error:
**starting generator
Error: Could not find file '&lt;path&gt;\\# A set of common base type definitions'.**
This is weird, but not so weird when I check the contents of the first included file _requests.raml_:
The first operative line in that included file also tries to include the baseTypes.raml file so:
```
uses:
  Base: baseTypes.raml      # A set of common base type definitions
```
For some reason the parser is not exc;luding the comment and treating it as part of an included file path.

**Comment by ageoghegan** [comment]
Any chance you might rebuild the command line tools with this change in it?  I don't see the source for the actual command line version so I presume rebuilding it myself is tricky.

**Comment by woodp** [comment]
It's not difficult, just use https://github.com/mulesoft-labs/raml-dotnet-tools/blob/master/RAMLGen.sln with VS 2013.
The only tricky part with current code (dev branch) is that it's pointing to RAML.Parser 1.0.6 which is not yet released, so you need to change that back to 1.0.5 before building.
